src/components/LimitBar/LimitBar.tsx

```diff
@@ -75,26 +74,26 @@ export const LimitBar: FC<LimitBarProps> = ({
       <span>
         <b>{formatNumeric(quantity)}</b>
         {isUnlimited(limit) ? (
-          <span> / &#8734;</span>
+          <span> / &#8734; </span>
         ) : (
           `/${formatNumeric(limit)} `
         )}
-        {shouldShowRegularLimit && (
```

Comment:
Założenia były takie, że jeśli user ma Free Trial, to ta liczba w nawiasie (regularLimit) oraz tooltip obok niej (tooltipInQuantity) mają się pokazać. Jeśli dobrze rozumiem, zmiana tego komponentu spowoduje, że user może mieć regularLimit a nie musi widzieć tooltipa (co może powodować problem, bo ten tooltip jest dedykowany tej liczbie i tłumaczy, co ona w ogóle znaczy) oraz na odwrót. No i będzie widział to każdy user, nie tylko ten na free trial'u, a jeśli nie będzie na free trialu, to nie wiem co wtedy zwraca backend, czy 0 czy co, a tą zwrotkę zobaczy użytkownik

Reply:
założenia mam na myli [GRO-4198](https://landingi.atlassian.net/jira/software/projects/GRO/boards/76?selectedIssue=GRO-4198)

Reply:
Omówimy temat w poniedziałek. Moim zdaniem można to ograć bez tego konkretnego propsa. 
Można go też zostawić, ale musimy zrobić to bardziej restrykcyjnie. 
Nic na razie nie wrzucam i przy niczym się nie upieram ;)

Reply:
Jasne, nawet można jakiegoś calla zgrać, bo ja sam mam parę pytań, jak to powinniśmy zrobić :)